wsl/accessibility/ubuntu-wsl-doc-images.md

```diff
@@ -41,123 +41,126 @@ explain what it is and what it does:
 
-![|541x276](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/splash-screen.png?raw=true)
+![NVIDIA Graphics Driver startup page](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/splash-screen.png?raw=true)
 
-![|542x412](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/license.png?raw=true)
+![NVIDIA software license agreement](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/license.png?raw=true)
 
-![|536x397](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/installation-options.png?raw=true)
+![NVIDIA Graphics Driver installation options with "Express" checked](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/installation-options.png?raw=true)
 
-![|542x404](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/installing.png?raw=true)
+![NVIDIA Virtual Host controller installation progress](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/installing.png?raw=true)
 
-![|544x406](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/install-finished.png?raw=true)
+![NVIDIA Graphics Driver installation success page](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/install-finished.png?raw=true)
 
-![|544x559](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/done-done.png?raw=true)
+![Terminal output showing successful installation of NVIDIA CUDA toolkit on Ubuntu](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/done-done.png?raw=true)
 
-![|624x135](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/make.png?raw=true)
+![Terminal output showing the successful compilation of a CUDA sample application](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/make.png?raw=true)
 
-![|548x599](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/device-query.png?raw=true)
+![Terminal output showing the results of running the deviceQuery sample application](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/gpu-cuda/device-query.png?raw=true)
 
 ## Use WSL for data science and engineering
 
 [link to documentation page](https://canonical-ubuntu-wsl.readthedocs-hosted.com/en/latest/tutorials/data-science-and-engineering/)
 
-![|624x373](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/data-science-engineering/octave.png?raw=true)
+![Windows desktop showing a terminal running "octave --gui" and the Octave GUI](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/data-science-engineering/octave.png?raw=true)
 
-![|570x225](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/data-science-engineering/save-file.png?raw=true)
+![Octave GUI showing the "Save and Run button" for the juliatest.m file](https://github.com/ubuntu/WSL/blob/main/docs/tutorials/assets/data-science-engineering/save-file.png?raw=true)
```

Comment:
See above comment. I think graphical interface is better here.

Also, I don't think we need to include the file extension in the alt text - it has already been stated in the text and it should be obvious what "juliatest file" refers to.

Reply:
Done.